src/sentry/assistant/guides.py

```diff
@@ -52,157 +49,55 @@
-                'message': _(
-                    'This is a unique identifier for the issue and can be included in a commit '
-                    'message to tell Sentry to resolve the issue when the commit gets deployed. '
-                    'See <a href="https://docs.sentry.io/learn/releases/" target="_blank">Releases</a> '
-                    'to learn more.'),
-                'target': 'issue_number',
-            },
-            {
-                'title': _('Issue Tracking'),
+                'title': _('Delete and Ignore'),
                 'message': _(
-                    'Create issues in your project management tool from within Sentry. See a list '
-                    'of all integrations <a href="https://docs.sentry.io/integrations/" target="_blank">here</a>.'),
-                'target': 'issue_tracking',
+                    'Delete an issue to remove it from your issue list until it happens again. '
+                    'Ignore an issue to remove it permanently or until certain conditions are met.'),
+                'target': 'ignore_delete_discard',
             },
             {
-                'title': _('Ignore, Delete and Discard'),
+                'title': _('Issue Number'),
                 'message': _(
-                    'Ignoring an issue silences notifications and removes it from your feeds. '
-                    'Deleting an issue deletes its data and causes a new issue to be created if it '
-                    'happens again. Delete & Discard (available on the medium plan and higher) '
-                    'deletes most of the issue\'s data and discards future events matching the '
-                    'issue before they reach your stream. This is useful to permanently ignore '
-                    'errors you don\'t care about.'),
-                'target': 'ignore_delete_discard',
+                    'Include this unique identifier in your commit message to have Sentry automatically '
+                    'resolve the issue when your code is deployed. '
+                    '<a href="https://docs.sentry.io/learn/releases/" target="_blank">Learn more</a>.'),
+                'target': 'issue_number',
             },
             {
-                'title': _('Issue Owners'),
+                'title': _('Ownership Rules'),
                 'message': _(
-                    'Define users or teams that are responsible for specific paths or URLS so '
-                    'that notifications can be routed to the correct people. Learn more '
-                    '<a href="https://docs.sentry.io/learn/issue-owners/" target="_blank">here</a>.'),
+                    'Define users or teams responsible for specific file paths or URLs so '
+                    'that alerts can be routed to the right person. '
+                    '<a href="https://docs.sentry.io/learn/issue-owners/" target="_blank">Learn more</a>.'),
                 'target': 'owners',
             },
         ],
     },
-    'releases': {
```

Comment:
Did we remove a bunch of tips, or were these just not needed anymore?

Reply:
yes these seemed unhelpful. e.g. we have empty state in releases that is much better than this.